tests/e2e/specs/admin/mobile-redirect.js

```diff
@@ -23,7 +23,8 @@ describe( 'Mobile redirect settings', () => {
 		await expect( page ).toMatchElement( `${ toggleSelector }:checked` );
 	} );
 
-	it( 'persists the mobile redirect setting off', async () => {
+	// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Any reason for skipping this test?

Reply:
Yeah, I was having trouble getting Puppeteer to click the mobile toggle, but I see this morning it was because the toggle was hiding behind the fixed navbar. Resolved (and unskipped the test) in 330343727